sdk/extensions/Microsoft.Extensions.Azure/src/AzureClientServiceCollectionExtensions.cs

```diff
@@ -28,12 +29,36 @@ public static void AddAzureClients(this IServiceCollection collection, Action<Az
 
         /// <summary>
         /// Adds the minimum essential Azure SDK interop services like <see cref="AzureEventSourceLogForwarder"/> and <see cref="AzureComponentFactory"/> to the specified <see cref="IServiceCollection"/> without registering any client types.
+        /// Azure SDK logging will not be enabled by default, but can be enabled by calling the <see cref="AzureEventSourceLogForwarder.Start"/> method.
+        /// Alternatively, you can use the <see cref="AddAzureClientsCore(Microsoft.Extensions.DependencyInjection.IServiceCollection, bool)"/> overload
+        /// and pass <value>true</value> to enable logging.
         /// </summary>
         /// <param name="collection">The <see cref="IServiceCollection"/>.</param>
         public static void AddAzureClientsCore(this IServiceCollection collection)
+        {
+            collection.AddAzureClientsCore(false);
+        }
+
+        /// <summary>
+        /// Adds the minimum essential Azure SDK interop services like <see cref="AzureEventSourceLogForwarder"/> and <see cref="AzureComponentFactory"/> to the specified <see cref="IServiceCollection"/> without registering any client types.
+        /// </summary>
+        /// <param name="collection">The <see cref="IServiceCollection"/>.</param>
+        /// <param name="enableLogging">Whether or not to enable Azure SDK logging. Even if this is set to <value>false</value>, logging can be enabled
+        /// by calling the <see cref="AzureEventSourceLogForwarder.Start"/> method.</param>
+        public static void AddAzureClientsCore(this IServiceCollection collection, bool enableLogging)
```

Comment:
Should we consider `startLogging`?   Isn't logging "enabled" either way - this is just about whether we automatically start the forwarder or require the someone do it manually, no?

Reply:
The logging would only be enabled if someone manually starts the log forwarder. Otherwise they will not get any logs.

Reply:
What about enableLogForwarding?

Reply:
oohh... I like that!